src/views/mixed-content/hsts-percentage-requests/elements.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const ChartContainer = styled.div`
+  height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center
```

Comment:
Done!

Reply:
What I'm looking is that every component folder has the same file with the same contents, is that supposed to be that way @ernestognw ? or can we refactor that?

Reply:
You're correct Jonathan. At this point, every container is almost the same, but that doesn't mean they're equal. Once layout is refactored, every ChartContainer will vary, so even if its styles are the same, their context and application are different.

Later we can create a folder with templates with repeated logic, but it is too early to do it since we don't really know what the patterns are yet.